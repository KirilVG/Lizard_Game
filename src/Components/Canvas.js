import React from "react";
import * as myConstants from "./Constants";

const Canvas = (props) =>{
    return (
        <canvas id={props.canvasID} width={window.innerWidth} height={window.innerHeight * myConstants.heightOffset} />
    );
}

export default Canvas;
