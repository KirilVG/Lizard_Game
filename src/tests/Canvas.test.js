import renderer from "react-test-renderer";
import Canvas from "../Components/Canvas";

const mockedId = "Id";

it("canvas should be rendered correctly", () => {
  const tree = renderer.create(<Canvas canvasID={mockedId} />).toJSON();
  expect(tree).toMatchSnapshot();
});
