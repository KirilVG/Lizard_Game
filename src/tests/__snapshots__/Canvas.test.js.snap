// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`canvas should be rendered correctly 1`] = `
<canvas
  height={760.3199999999999}
  id="Id"
  width={1024}
/>
`;
